pkg/cache/clusterqueue.go

```diff
@@ -230,7 +233,9 @@ func (c *clusterQueue) updateQueueStatus() {
 		len(c.missingAdmissionChecks) > 0 ||
 		len(c.inactiveAdmissionChecks) > 0 ||
 		len(c.multipleSingleInstanceControllersChecks) > 0 ||
-		len(c.flavorIndependentAdmissionCheckAppliedPerFlavor) > 0 {
+		len(c.flavorIndependentAdmissionCheckAppliedPerFlavor) > 0 ||
```

Comment:
was this check for `flavorIndependentAdmissionCheckAppliedPerFlavor` missing before?

Reply:
no, it was there what changed is just `||`

Reply:
ah, I missed that